client/client.go

```diff
@@ -16,21 +16,60 @@ package client
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/tigrisdata/tigrisdb-client-go/driver"
 )
 
+// Client is the interface for a TigrisDB client.
 type Client interface {
+	// Database returns a Database interface for the provided
```

Comment:
For Java interface I added listDBs, create/drop DB in the client interface. Where are you thinking of placing them?

Reply:
Same place! I put create in here, we can add the others